moveit_core/planning_scene/src/pyplanning_scene.cpp

```diff
@@ -0,0 +1,96 @@
+                                               const kinematic_constraints::KinematicConstraintSet&,  //
+                                               bool>(&PlanningScene::isStateConstrained, py::const_);
+auto IS_STATE_CONSTRAINED2 = py::overload_cast<const moveit::core::RobotState&,                       //
+                                               const kinematic_constraints::KinematicConstraintSet&,  //
+                                               bool>(&PlanningScene::isStateConstrained, py::const_);
+auto IS_STATE_CONSTRAINED3 = py::overload_cast<const moveit_msgs::RobotState&,   //
+                                               const moveit_msgs::Constraints&,  //
+                                               bool>(&PlanningScene::isStateConstrained, py::const_);
+auto IS_STATE_CONSTRAINED4 = py::overload_cast<const moveit::core::RobotState&,  //
+                                               const moveit_msgs::Constraints&,  //
+                                               bool>(&PlanningScene::isStateConstrained, py::const_);
+
+auto GET_TRANSFORMS1 = py::overload_cast<>(&PlanningScene::getTransforms, py::const_);
+
+auto IS_STATE_VALID1 = py::overload_cast<const moveit_msgs::RobotState&,  //
+                                         const std::string&,              //
+                                         bool>(&PlanningScene::isStateValid, py::const_);
+auto IS_STATE_VALID2 = py::overload_cast<const moveit::core::RobotState&,  //
+                                         const std::string&,               //
+                                         bool>(&PlanningScene::isStateValid, py::const_);
+auto IS_STATE_VALID3 = py::overload_cast<const moveit_msgs::RobotState&,   //
+                                         const moveit_msgs::Constraints&,  //
+                                         const std::string&,               //
+                                         bool>(&PlanningScene::isStateValid, py::const_);
+auto IS_STATE_VALID4 = py::overload_cast<const moveit::core::RobotState&,  //
+                                         const moveit_msgs::Constraints&,  //
+                                         const std::string&,               //
+                                         bool>(&PlanningScene::isStateValid, py::const_);
+auto IS_STATE_VALID5 = py::overload_cast<const moveit::core::RobotState&,                       //
+                                         const kinematic_constraints::KinematicConstraintSet&,  //
+                                         const std::string&,                                    //
+                                         bool>(&PlanningScene::isStateValid, py::const_);
+
+auto SET_CURRENT_STATE1 = py::overload_cast<const moveit_msgs::RobotState&>(&PlanningScene::setCurrentState);
+auto SET_CURRENT_STATE2 = py::overload_cast<const robot_state::RobotState&>(&PlanningScene::setCurrentState);
```

Comment:
Do you really need both variants of methods (msg and C++ class) for this and all the above methods?
I think, the C++ class is converted to a msg anyway (via casters), isn't it?